src/modules/getmovies.js

```diff
@@ -0,0 +1,190 @@
+import { getComments, getCommentCount } from './comments.js';
+import updateTotalLikes from './likes.js';
+
+const countMovies = (data) => data.length;
+
+const initializeComments = (data) => {
+  const commentButtons = document.querySelectorAll('.comment');
+
+  commentButtons.forEach((button) => {
+    button.addEventListener('click', () => {
+      const popup = document.querySelector('#popup-modal');
+      const movieId = button.getAttribute('data-id');
+      const relatedItem = data.find((item) => item.id === parseInt(movieId, 10));
+
+      popup.innerHTML = '';
+
+      const popupMarkup = `
+
+
+        <div class="img">
+          <div class="movies-details-section">
+            <img src="${relatedItem.image.original}" alt="movies images">
+            <span id="close-btn">&#10005;</span>
+            <div class="summary">
```

Comment:
- [ ] (Optional). It would be nice if your movie cards matched the given wireframe, this increases employers confidence in your abilities to follow design guidelines

Your cards
![image](https://github.com/ANTOINE1128/javascript-capstone/assets/81932478/5557db82-cd72-4bd5-a055-7cf97cc338a7)

Given cards
![image](https://github.com/ANTOINE1128/javascript-capstone/assets/81932478/15d5c9f4-740b-4ced-9393-46e18bc903cf)

#### Note the position of the like icon in your design